src/containers/Post/PostDetailed.js

```diff
@@ -27,6 +27,9 @@ class PostDetailed extends Component {
             <p>
               <b>Text:</b> {post.text}
             </p>
+            <p>
+              <b>Created:</b> {new Date(post.dateCreated).toLocaleString()}
```

Comment:
Check out MDN docs for these kind of methods, some of them are not really supported in all relevant browsers.

Reply:
it seems to be supported (basic support in all browsers)